proposals/raster_to_vector.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "raster_to_vector",
+    "summary": "Converts a raster to a vector",
+    "description": "Converts a raster data cube into a vector data cube. The bounding polygon of similar areas of pixels is constructed.",
```

Comment:
What does "similar" mean? It suggests there is some room for small deviations, but then there should be a parameter to set thresholds I think. Or a parameter to set the number of desired quantization levels.

Reply:
Another related question: how are bands handled here? Is each band converted separately, so you have different vector segmentation for each band? If so, I don't think that can represented as a single openEO vector cube, because the geometries are the labels of the "geometry" dimension, so the output would be a set of vector cubes, one for each input band.

Or does segmentation happens on all bands simultaneously,  e.g. RGB-color based segmentation? But then the definition of "similar" or color quantization becomes a lot harder I guess.

Maybe raster_to_vector should only be allowed on single band raster cubes?

Reply:
Similar refers to uniform or homogenous. I'll use a uniform in this case for clarity. 

The concerns on how to deal with multiple bands are legit. I have not come across any platform that does that and hence the conversion of a single band raster cube seems ideal.

Reply:
If you really mean that pixel values should be the same, I would just state that, e.g.

> The bounding polygon of areas with equal pixel values is constructed.

However, this works probably good enough when your input data is integer valued (e.g. classification values),
but when it's float data (e.g. NDVI values), a user probably wants to to define quantization in some way. Is this in scope of this PR, or is that not necessary yet?